src/services/token.service.js

```diff
@@ -0,0 +1,24 @@
+const { Token } = require('../models/token.model');
+
+const getByToken = async (refreshToken) => {
+  return Token.findOne({ where: { refreshToken } });
```

Comment:
Consider adding error handling for the `Token.findOne` operation to manage potential errors and prevent unhandled promise rejections.